src/components/resources/Resources.js

```diff
@@ -65,26 +64,28 @@ export default function Home() {
         annouced that Covid-19 is considered a pandemic. Many countries have
         issued stay at home orders or recommendations to quarantine and self
         isolate.
-        <br />
-        <br />
-        Because people find it hard to stay at home alone,{" "}
+      </p>
+      <p>
+        Because people find it hard to stay at home alone,
         <a
           target="_blank"
           rel="noreferrer"
           href="https://apnews.com/article/31e3e60e7ea6bc4566b0ee3998ab98a6"
         >
+          {" "}
           a lot of people decided to adopt a friend
         </a>
         .
-        <br />
-        <br />
-        <b>
+      </p>
+      <p>
+        <span>
           Pawternity Hub wants to help you stay safe and help you find a friend
           that can keep you company during the pandemic
-          <br /> Check out the links below for more information about adoption
-          during Covid-19
-        </b>
-        <br />
+          <p>
+            Check out the links below for more information about adoption during
+            Covid-19
+          </p>
```

Comment:
Lets remove the p tags here. Was thinking instead of a having a span tag there, we could have a `className="resource__paragraph-bold"` in the ptag at line 80 and in the css `font-weight:bold`

```
.resource__paragraph-bold {
font-weight: bold;

}

Reply:
So, I should add a class name to the `<p>` tag at line 80 alone? What about the other **bolded** paragraphs?

Reply:
oh u can add it them too

Reply:
Done